packages/test/test-end-to-end-tests/src/test/deRehydrateContainerTests.spec.ts

```diff
@@ -468,8 +468,8 @@ describeFullCompat(`Dehydrate Rehydrate Container Test`, (getTestObjectProvider)
             const sharedStringBefore = await defaultDataStoreBefore.getSharedObject<SharedString>(sharedStringId);
             const intervalsBefore = sharedStringBefore.getIntervalCollection("intervals");
             sharedStringBefore.insertText(0, "Hello");
-            let interval0 = intervalsBefore.add(0, 0, IntervalType.SlideOnRemove);
-            let interval1 = intervalsBefore.add(0, 1, IntervalType.SlideOnRemove);
+            let interval0: SequenceInterval | undefined = intervalsBefore.add(0, 0, IntervalType.SlideOnRemove);
```

Comment:
`IntervalCollection.add` cannot be undefined; however, later in this function `interval0` is assigned to `IntervalCollection.getIntervalById`, which _can_ be undefined

Reply:
fair enough. FWIW I stylistically prefer just using a different variable and also functions which typeguard (sad our assert library doesn't) to avoid this, but since it's in a test not a big deal.